apps/docs/src/app/platform/component-docs/platform-list/platform-list-examples/platform-list-with-delete-button-example.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, HostListener } from '@angular/core';
+
+@Component({
+    selector: 'fdp-list-with-delete-button-example',
+    templateUrl: './platform-list-with-delete-button-example.component.html'
+})
+export class PlatformListWithDeleteButtonExampleComponent {
+    items: any[] = [
+        { 'title': 'Item1', 'secondayIcons': [{ 'icon': 'decline', 'isButton': 'true' }] },
+        { 'title': 'Item2', 'secondayIcons': [{ 'icon': 'decline', 'isButton': 'true' }] },
+        { 'title': 'Item3', 'secondayIcons': [{ 'icon': 'decline', 'isButton': 'true' }] },
+        { 'title': 'Item4' }
+    ];
+
+    // Handle deletion of item via  mouseclick
+    @HostListener('click', ['$event'])
+    deleteRow(event: any) {
+        if (event.target.parentNode.tagName.toLowerCase() === 'fdp-button') {
+            event.target.parentNode.parentNode.parentNode.parentNode.parentNode.style.display = 'none';
```

Comment:
shouldn't this logic be coming from the lib?

Reply:
list item will provide options for actions, like delete, edit etc, but on click of it what exactly. needs to. be done that is up to the application team, they are either remove the item from the array, JSON, or in backend DB or can hide it as shown in the above example.